content/en/docs/instrumentation/java/manual.md

```diff
@@ -190,22 +190,39 @@ You should get a list of 12 numbers in your browser window, for example:
 For both library and app instrumentation, the first step is to install the
 dependencies for the OpenTelemetry API.
 
+### Dependency management
+
+A Bill of Material
+([BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms))
+ensures that versions of dependencies (including transitive ones) are aligned.
+Importing the `opentelemetry-bom` BOM is important to ensure version alignment
+across all OpenTelemetry dependencies.
+
 {{< tabpane text=true >}} {{% tab Gradle %}}
 
-```kotlin { hl_lines=3 }
+```kotlin { hl_lines=["1-5",9] }
+	  imports {
+		    mavenBom("io.dependencyManagement {
+opentelemetry:opentelemetry-bom:{{% param vers.otel %}}")
+	  }
+}
+
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web");
-    implementation("io.opentelemetry:opentelemetry-api:{{% param vers.otel %}}");
+    implementation("io.opentelemetry:opentelemetry-api");
 }
 ```
 
-Throughout this documentation you will add dependencies. For a full list of
-artifact coordinates, see [releases]. For semantic convention releases, see
```

Comment:
> Throughout this documentation you will add dependencies. For a full list of
artifact coordinates, see [releases].

I like this section near the top - maybe even the first line in the dependency management section. Starting with the bom feels strange without first establishing which artifacts the bom aligns.